src/components/home/Modal.tsx

```
@@ -1,13 +1,14 @@
-import { useEffect } from "react";
-import { createPortal } from "react-dom";
+import { useEffect, useState } from "react";
 import Character from "@/assets/images/character.png";
+import Modal from "../common/Modal";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)